apps/site/src/app/events/schemas/LocationSchema.ts

```diff
@@ -18,7 +18,7 @@ const RegionSchema = createEnumSchema(validRegionIds)
 export const LocationSchema = z
   .object({
     primary: z.string(),
-    region: RegionSchema.optional(),
+    region: RegionSchema.nullable().optional(),
```

Comment:
Not sure if that reads better, but this is equivalent to [nullish](https://zod.dev/?id=nullish)

```suggestion
    region: RegionSchema.nullish(),
```